app/views/landing-pages/roblox/PageRoblox.vue

```diff
@@ -1,148 +1,111 @@
-          <p v-if="role==='partner'" v-html="$t('roblox_landing.bottom_blurb_partner', i18nData)"></p>
+    <div class="container-fluid container-fluid-boxes">
+      <div v-if="role" class="container container-boxes">
+        <h3>{{ $t('roblox_landing.boxes_title') }}</h3>
+        <div v-for="boxType in boxesByRole[role]" class="row" :class="`row-type-${boxType}`">
+          <div class="col col-md-6 box-content ">
+            <img class="box-icon" :src="`/images/pages/roblox/${boxType}-icon.svg`">
+            <h4 class="box-title">
+              {{ $t(`roblox_landing.box_${boxType}_subhead`) }}
+            </h4>
+            <p>
+              {{ $t(`roblox_landing.box_${boxType}_blurb_${role}`) }}
+            </p>
+          </div>
+          <div class="col col-md-6">
+            <img :src="`/images/pages/roblox/${boxType}.png`">
+          </div>
         </div>
-      </div>
-    </div>
-
-    <div class="container">
-      <div class="row row-video">
-        <div class="col-md-12">
-          <div class="video-container">
-            <base-video
-                :youtube-props="{ videoId: youtubeId, fitParent: true }"
-                :cloudflare-props="{ videoCloudflareId: videoId, thumbnailUrl }"
-            />
+        <div class="row">
+          <div class="col col-md-12">
+            <p v-if="role === 'parent'" v-html="$t('roblox_landing.bottom_blurb_parent', i18nData)"></p>
+            <p v-if="role === 'partner'" v-html="$t('roblox_landing.bottom_blurb_partner', i18nData)"></p>
           </div>
         </div>
       </div>
 
-      <div class="row row-faq">
-        <div class="col-md-12">
-          <button-main :href="false" :buttonText="$t('contact.faq')" class="button-main"/>
-
-          <div class="item">
-            <p class="question">{{ $t('roblox_landing.question_1') }}</p>
-            <ul>
-              <li>{{ $t('roblox_landing.answer_1') }}</li>
-            </ul>
-          </div>
-
-          <div class="item">
-            <p class="question">{{ $t('roblox_landing.question_2') }}</p>
-            <ul>
-              <li>{{ $t('roblox_landing.answer_2') }}</li>
-            </ul>
+      <div class="container">
+        <div class="row row-video">
+          <div class="col-md-12">
+            <div class="video-container">
+              <base-video :youtube-props="{ videoId: youtubeId, fitParent: true }"
+                :cloudflare-props="{ videoCloudflareId: videoId, thumbnailUrl }" />
+            </div>
           </div>
+        </div>
 
-          <div class="item">
-            <p class="question">{{ $t('roblox_landing.question_3') }}</p>
-            <ul>
-              <li>{{ $t('roblox_landing.answer_3') }}</li>
-            </ul>
+        <div class="row row-faq">
+          <div class="col-md-12">
+            <button-main :href="false" :buttonText="$t('contact.faq')" class="button-main" />
```

Comment:
This isn't new, but noticed that this is a button that doesn't do anything. We should probably use some other element instead of a button. (Or is it intended to do something?)